src/solvers/refinement/string_refinement.cpp

```diff
@@ -777,62 +1046,73 @@ std::string string_refinementt::string_of_array(const array_exprt &arr)
 
 /// Display part of the current model by mapping the variables created by the
 /// solver to constant expressions given by the current model
-void string_refinementt::debug_model()
+void debug_model(
```

Comment:
`super_get` and `max_string_length` can be `const`.

Reply:
Arguments have been made const (it would be helpful if those comments were just below those non-const arguments, rather than at the top of the function).